.github/workflows/phar-build.yml

```
@@ -0,0 +1,60 @@
+          ./conduit.phar list
+
+      - name: Upload PHAR as Artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: conduit-phar
+          path: conduit.phar
+          retention-days: 30
+
+      - name: Upload PHAR to Release
+        if: startsWith(github.ref, 'refs/tags/')
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)